ui/components/component-library/avatar-account/README.mdx

```diff
@@ -0,0 +1,21 @@
+import { Story, Canvas, ArgsTable } from '@storybook/addon-docs';
+
+import { AvatarAccount } from './avatar-account';
+
+# AvatarAccount
+
+The `AvatarAccount` is a type of avatar reserved for representing accounts.
+
+<Canvas>
+  <Story id="ui-components-component-library-avatar-account-avatar-account-stories-js--default-story" />
+</Canvas>
+
+## Props
+
+The `AvatarAccount` accepts all props below 
+
+<ArgsTable of={AvatarAccount} />
+
+### children
+
+The children to be rendered inside the AvatarAccount. It could either be the `Jazzicon` or `Blockie` component
```

Comment:
No longer used?

```suggestion
```

Reply:
Not required since we replaced children with the component itself

Reply:
Oh my suggestion is to remove the documentation about the children if we aren't using it? It looks like it still exists?